plugins/chat/assets/javascripts/discourse/models/chat-message.js

```diff
@@ -69,6 +70,7 @@ export default class ChatMessage {
     this.hidden = args.hidden || false;
     this.chatWebhookEvent = args.chatWebhookEvent || args.chat_webhook_event;
     this.createdAt = args.createdAt || args.created_at;
+    this.deletedById = args.deleted_by_id;
```

Comment:
```suggestion
    this.deletedById = args.deletedById || args.deleted_by_id;
```

Reply:
That actually shouldn't be needed (I know we have it just above), it shows something wrong if it is, but it doesn't hurt for now :+1: